core/scripts/sql/texera_ddl.sql

```diff
@@ -163,16 +163,17 @@ CREATE TABLE IF NOT EXISTS file_of_workflow
 
 CREATE TABLE IF NOT EXISTS workflow_executions
 (
-    `eid`             INT UNSIGNED AUTO_INCREMENT NOT NULL,
-    `vid`             INT UNSIGNED NOT NULL,
-    `uid`             INT UNSIGNED NOT NULL,
-    `status`          TINYINT NOT NULL DEFAULT 1,
-    `result`          TEXT, /* pointer to volume */
-    `starting_time`   TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-    `last_update_time`   TIMESTAMP,
-    `bookmarked`      BOOLEAN DEFAULT FALSE,
-    `name`				VARCHAR(128) NOT NULL DEFAULT 'Untitled Execution',
+    `eid`                    INT UNSIGNED AUTO_INCREMENT NOT NULL,
+    `vid`                    INT UNSIGNED NOT NULL,
+    `uid`                    INT UNSIGNED NOT NULL,
+    `status`                 TINYINT NOT NULL DEFAULT 1,
+    `result`                 TEXT, /* pointer to volume */
+    `starting_time`          TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    `last_update_time`       TIMESTAMP,
+    `bookmarked`             BOOLEAN DEFAULT FALSE,
+    `name`				     VARCHAR(128) NOT NULL DEFAULT 'Untitled Execution',
     `environment_version`    VARCHAR(128) NOT NULL,
+    `log_location`           TEXT, /* uri to log storage */
```

Comment:
if changing the ddl, please add the `ddl-change` tag on the PR, and also provide an updated SQL script for deployment update.

Reply:
done